Task_DEV_4/Task_DEV_4/Parallelogram.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+namespace Task_DEV_4
+{
+    class Parallelogram : Square
+    {
+        public double HeldHeight { get; set; }
+        public double Angle { get; set; }
+        public double Side { get; set; }
```

Comment:
Maybe it would be better to rename it to Width like in Rectangle class.

Reply:
Done.